bug-bounty-list/README.md

```diff
@@ -0,0 +1,57 @@
+
+## Before you go hunting for new programs to add...
+
+Programs on bug-bounty-list need to satisfy the definition of a public bug bounty or vulnerability disclosure program, which means they need two key components:
+  1. A publicly accessible vulnerability disclosure policy sometimes called a program brief, or bounty brief, and 
+  2. A publicly accessible channel for vulnerability submission **which is explicitly mentioned in the vulnerability disclosure policy**.  
+  
+Some examples of suggestions that won't be accepted:
+  1. Private programs  
+  2. Invitation-only programs  
+  3. Programs that you heard about from a friend but aren't listed publicly  
+  4. Programs without a public policy or a nominated channel for communication  
+
+## Definition of fields 
+
+**program_name (required):** What is the name of the company or product team responsible for the program?  
+**policy_url (required):** Where can the policy which defines the program be found? (Usually a webpage e.g. /security, a URL on a vulnerability management platform, or in a robots.txt or security.txt file).  
+**submission_url (required):** Where does policy say vulnerability submissions should be sent? (Usually a web form, a URL on a vulnerability management platform, or an email address. Emails can be added as URLs using 'mailto:').  
+**launch_date (optional):** When does the program say it was launched?  
+**cash_reward (optional):** Does the program commit to rewarding vulnerabilities?  
+**swag (optional):** Does the program commit to sending swag out for vulnerabilities?  
+**hall_of_fame (optional):** Does the program have a listing or Hall of Fame page acknowledging vulnerability reporters . 
+**safe_harbor (required):** Does the program support full or partial safe-harbor?  
+
+~~~~
+  {
+    "program_name": "Tesla",
+    "policy_url": "https://bugcrowd.com/tesla",
+    "submission_url": "https://bugcrowd.com/tesla/report",
+    "launch_date": "",
+    "cash_reward": true,
+    "swag": false,
+    "hall_of_fame": true,
+    "safe_harbor": "full"
+  },
+~~~~
+
+*Example: Tesla... Can you find the date they first launched their bug bounty program?*
+
+## How to fill out the safe-harbor field
+
+The list supports three safe-harbor states: Full, Partial, and None. The flow below will help you determine which applies to the policy as you contribute to the list:  
+
+1. Does the policy mention anything about legal protections for good-faith hackers? If it does, go to Step 2. If not, then you should choose **"None"**. 
+2. Does the policy talk about authorization of testing and/or exemption from legal terms? If it does, you should choose **"Full"**. If not got to Step 3.
```

Comment:
I guess my previous comment may be irrelevant if you're going to restate these terms here, but I'd use similar language to avoid conflicting definitions (or rewrite the original definition in the parent readme).

"Partial" should be used when the Disclose.io requirements are _not_ met but the program still has a statement about authorizing good-faith testing, and "Full" should only be used if the Disclose.io requirements are met alongside such an authorization statement.